clusters.cnp.example.org/composition.yaml

```
@@ -5,10 +5,10 @@ metadata:
 spec:
   writeConnectionSecretsToNamespace: crossplane-system
   reclaimPolicy: Delete
-  compositeResource:
+  compositeType:
```

```suggestion
  compositeTypeRef:
```

@muvaf What do you think about this? I could interpret API conventions as this being valid - it's a reference to a thing (a type) and is a subset of ObjectReference.

---

That sounds better.